hr_shift/models/shift_planning.py

```diff
@@ -322,7 +322,7 @@ class ShiftPlanningLine(models.Model):
     end_hour = fields.Float(string="End hour", related="template_id.end_time")
     color = fields.Integer(related="template_id.color")
     planning_id = fields.Many2one(related="shift_id.planning_id")
-    employee_id = fields.Many2one(related="shift_id.employee_id")
+    employee_id = fields.Many2one(related="shift_id.employee_id", store=True)
```

Comment:
Instead of this, you can put auto_join=True for not having to store and don't affect performance.

Reply:
I have defined it in the same way as the `resource_id` field, defining it with `auto_join=True` is not enough, because when I try to group by that field I get the following error:

`Field hr.shift.planning.line.employee_id is not a stored field, only stored fields (regular or many2many) are valid for the 'groupby' paramete`